lib/foreman_tasks_core/task_launcher/abstract.rb

```diff
@@ -13,6 +13,8 @@ def launch!(_input)
         raise NotImplementedError
       end
 
+      def self.input_format; end
```

Comment:
What's purpose of this method?

Reply:
I used it as a reminder of how the input for the runner will look like (see [here](https://github.com/theforeman/foreman-tasks/pull/386/files#diff-186c5dc3b3701e52c768b38bea0c1c2cR5)), guess all these could be removed